server.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express')
+const app = express()
+const port = 3001
+
+app.get('/hello', (req, res) => {
+    res.send('Hello World!')
+})
+
+app.listen(port, () => {
+    console.log(`listening on port ${port}!`)
+})
```

Comment:
Add a new line to the end of this file - that's what the little red circle warning is about. It's actually really important for files to end with a newline, but I can't remember why :P